### PR TITLE
Rat-18: Create color system

### PR DESCRIPTION
**Pull request by luanstavares**
### Important Links

- Task: https://rateio.atlassian.net/browse/RAT-18?atlOrigin=eyJpIjoiZDA2Y2YzNTNkMGU2NGM3MWE1Y2I1MWI1YzBjZjhkZjgiLCJwIjoiaiJ9

### Changes, decisions, observations

- Color variables were created to make our design decisions standardized.
- This branch also contains "RAT-19: Create brand logo". (It was miss-committed on this branch)

### Required Checklist

Before proceeding, mark the following fields as completed:

- [x] Application is running without issues in the development environment
- [x] Code is following Figma design patterns

### CODE REVIEW

File: components/Logo/Logo.module.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.logoSmall {
+  @extend .logo;
```
Comment (nicolaslima321) [inline components/Logo/Logo.module.scss:2]: We also can use **&.classname** inside of parent class, its like a "@extend" syntax sugar. But we can skip that for now

File: styles/variables.scss [hunk 2]
```diff
@@ -1,4 +1,58 @@
 // Color Variables
-$primary-color: #ff9900;
-$secondary-color: #20008d;
-$ternary-color: #dddddd;
+
+//Primary Colors
+$primary-color-50: #ff9900;
```
Comment (nicolaslima321) [inline styles/variables.scss:4]: congrats for the work man, I liked a lot :star_struck:

File: styles/variables.scss [hunk 3]
```diff
@@ -1,4 +1,58 @@
...
+
+$font-weights: (
+  "regular": 400,
+  "medium": 500,
+  "semi-bold": 600,
+  "bold": 700,
+);
+
+//Logo sizes
+$logoSizes: (
```
Comment (nicolaslima321) [inline styles/variables.scss:53]: lets standarize the hypenated (with the **-**) words for sass variables **"$some-name"**
```suggestion
$logo-sizes: (
```

File: styles/variables.scss [hunk 4]
```diff
@@ -1,4 +1,58 @@
...
+$font-body-small: 14px;
+$font-caption-large: 12px;
+$font-caption-small: 10px;
+
+$font-weights: (
+  "regular": 400,
+  "medium": 500,
+  "semi-bold": 600,
+  "bold": 700,
+);
```
Comment (nicolaslima321) [inline styles/variables.scss:50]: declaring the variable like that, if you want to use the "regular" you will use as **$font-weights-regular** ? awesome :D

**Review by nicolaslima321 — APPROVED** [review]

File: styles/variables.scss (hunk 2 above)
Comment (luanstavares) [reply styles/variables.scss:4]: Thank you! Let's go!! 🚀

File: styles/variables.scss (hunk 4 above)
Comment (luanstavares) [reply styles/variables.scss:50]: Yes, this is equivalent to an array in JS, you can map through the elements. :)

Comment (luanstavares) [reply styles/variables.scss:50]: The is a method called map-get(<array>, <key>). This will return the value of the variable inside the array 😃

**Pull request merged**